ImageJ/EPICS_areaDetector/EPICS_NTNDA_Viewer.java

```diff
@@ -19,6 +19,10 @@
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
```

Comment:
Is this line needed?  I don't see LinkOption used in the code?

Reply:
I think it can be removed. I don't remember why I added it...